### ISSUE TITLE
Feature Request

### ISSUE THREAD

**Issue by stancruzsolo** [MTACS/Lynx#172]
Feature requests
1. Ability to customise app icon labels on the springboard. (Artemis does this)
2. Ability to hide/show carrier name and set WiFi Network as carrier name. (Artemis does this)
3. Ability to disable spotlight search on springboard/lockscreen. (Artemis does this)
4. Ability to show live download percentage as the app downloads from App Store. (Miscsettings does this)

Thank you!

**Comment by MTACS** [comment]
What do you mean by customize app icon labels, like renaming? You can already hide and color them. You can already hide the carrier, and I'll add the option to set it to the wifi address in the next update. As for the spotlight, I'm working on it as it causes conflicts with other tweaks.

**Comment by stancruzsolo** [comment]
Yes, I meant renaming apps.. also consider app percent i.e. download percentage on spring board.. found on miscsettings.. Thanks a lot btw.
